lib/tasks/verify.js

```diff
@@ -149,23 +149,23 @@ module.exports.scssLint = function(gulp, config) {
 };
 
 /**
- * Run the JSHint gulp plugin.
+ * Run the ESLint gulp plugin.
  */
-module.exports.jsHint = function(gulp, config) {
+module.exports.esLint = function(gulp, config) {
 	config = config || {};
-	var configPath = config.jsHintPath || path.join(__dirname, '/../../config/jshint.json');
+	var configPath = config.esLintPath || path.join(__dirname, '/../../config/.eslintrc');
 
 	// Exclude files specified in config.excludeFiles
 	excludeFiles = excludeFiles.concat(config.excludeFiles || []);
 
 	var combinedStream = combine(
 		gulp.src(['**/*.js'].concat(excludeFiles)),
-		jshint(configPath),
-		jshint.reporter('default'),
-		jshint.reporter('fail')
+		eslint({
+			configFile: configPath
+		}),
+		eslint.format('compact', process.stdout),
+		eslint.failAfterError()
```

Comment:
hmm we sure we want this?

>         // To have the process exit with an error code (1) on
>        // lint error, return the stream and pipe to failOnError last.

if you're pulling this in as a dependency and running it from inside node this behaves very oddly…  is this how it worked in jshitn?